src/ml/test.ml

```diff
@@ -23,12 +23,13 @@ let parse_pp_test path =
   let vll_file = Platform.gen_name !Platform.output_path filename ".v.ll" in
   let dot_s = Platform.gen_name !Platform.output_path filename ".s" in
   let _ = Printf.fprintf stderr "Running llc on: %s\n%!" path in
-  try 
-    let _ = llc_parse path dot_s in
+  try
+    (* SAZ: turning off using llc for now -- LLVM v. 9 doesn't parse these *)
```

Comment:
Do we want to merge this change in master? If so we might to open an issue to resolve this later